cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/dns/HttpContactPointBootstrap.scala

```diff
@@ -88,11 +84,12 @@ private[dns] class HttpContactPointBootstrap(
 
   override def receive = {
     case Internal.ProbeNow() ⇒
-      log.debug("Probing {} for seed nodes...", probeRequest.uri)
+      val req = ClusterBootstrapRequests.bootstrapSeedNodes(baseUri)
+      log.debug("Probing {} for seed nodes...", req.uri)
 
       http
         .singleRequest(probeRequest)
-        .flatMap(_.entity.toStrict(settings.contactPoint.probeTimeout))
+        .flatMap(_.entity.toStrict(1.second))
```

Comment:
1 second sounds tight, and not configurable, is it so little data that it will never be slower?

Reply:
Ok will increase